templates/registration/password_change_form.html

```diff
@@ -36,7 +37,8 @@
 {% block content %}
 
 <form method="POST" class="form-signin">{% csrf_token %}
-    <h1 class="h3 mb-3 font-weight-normal">Change Password</h1>
+    <i class="fas fa-unlock-alt" style="font-size: 7em; color: orange; margin-left:150px;margin-bottom: 3%;" ></i>
```

Comment:
Don't write so many inline styles. You can create a new class and specify the properties there.

Also, the colour hex should be `#ff8c00`. Again, don't specify margin in %.